testing/jormungandr-integration-tests/src/non_functional/explorer.rs

```diff
@@ -33,7 +36,7 @@ pub fn test_explorer_is_in_sync_with_node_for_15_minutes() {
             .with_kes_update_speed(KesUpdateSpeed::new(43200).unwrap()),
```

Comment:
not your code, but if we could clean up this magic number would be good :)

Reply:
I think this number is not so magic? It is pretty clear what it is?